man/memdb_frame.Rd

```diff
@@ -16,7 +16,7 @@ src_memdb()
 \item{...}{A set of name-value pairs. Arguments are evaluated sequentially,
 so you can refer to previously created elements. These arguments are
 processed with \code{\link[rlang:quos]{rlang::quos()}} and support unquote via \code{\link{!!}} and
-unquote-splice via \code{\link{!!!}}. Use \verb{:=} to create columns that start with a dot.}
+unquote-splice via \code{\link{!!!}}. Use \code{:=} to create columns that start with a dot.}
```

Comment:
Rebuilt the docs out of habit, and I'm not sure why this happened...can revert it if you like

Reply:
It looks like you need to upgrade roxygen2 (but don't worry about it for this PR, I'll take care of it)